### ISSUE TITLE
executing script from ./ubuntu_directory/ not ".termux/tasker"

### ISSUE THREAD

**Issue by roshanravan** [termux/termux-tasker#29]
i want to first run Ubuntu chroot for Termux by executing this script: ./ubuntu_directory/start-ubuntu.sh and then execute hollywood command

what i did was using VI command creating a script (for example: script1.sh) and in vi editor tried writing ./ubuntu_directory/start-ubuntu.sh and /ubuntu_directory/start-ubuntu.sh and start-ubuntu.sh  hoping it run the start-ubuntu.sh script from ./ubuntu_directory/ but didn't work so far, so i am stock and not figured it up how to execute hollywood after that

to sum it up i want to execute "./ubuntu_directory/start-ubuntu.sh" and then "hollywood" and i want to use Termux:Task, all i know is Termux Tasker will run scripts in ".termux/tasker" so maybe there is a way to make a script in .termux/tasker to run script or as shortcut of the start-ubuntu.sh 
 in  ./ubuntu_directory/ ? and then how to execute "hollywood" command? maybe with argument field?! do i make sense?

**Comment by xeffyr** [comment]
You need to create launch script in `~/.termux/tasker` which will call whatever you need.
If you have installed Ubuntu with scripts from https://github.com/Neo-Oli/termux-ubuntu/blob/master/ubuntu.sh, then you should be able to pass necessary arguments to `start-ubuntu.sh` so it will launch your program instead of just bash shell.

**Comment by roshanravan** [comment]
i created a myscript.sh in ~/.termux/tasker 👍 
#!/bin/bash
./ubuntu_directory/start-ubuntu.sh
but didn't work

**Comment by xeffyr** [comment]
> `#!/bin/bash`

=> `#!/data/data/com.termux/files/usr/bin/bash`

Termux is not standard environment and shebangs here different. Within shell you are saved by `termux-exec` hook but outside (Tasker plugin) is not.

> ./ubuntu_directory/start-ubuntu.sh
but didn't work

Yes, in that variant it will not work. Use absolute paths here and call script with its interpreter. Example:
```
bash /data/data/com.termux/files/home/ubuntu_directory/start-ubuntu.sh
```

**Comment by roshanravan** [comment]
it actually worked, 
#!/data/data/com.termux/files/usr/bin/bash
bash /data/data/com.termux/files/home/ubuntu_directory/start-ubuntu.sh

so with that the Ubuntu chroot terminal opens but now how to implant "hollywood" command after that?

**Comment by xeffyr** [comment]
> how to implant "hollywood" command after that?

Try
```
bash /data/data/com.termux/files/home/ubuntu_directory/start-ubuntu.sh /usr/games/hollywood
```

**Comment by roshanravan** [comment]
You are so Awesome! that was so cool, the good thing is now i can also make a Termux Widget and shortcut too, i am going to write a tutorial for running Hollywood Terminal with one click later
